stattest/experiment/test/test_step.py

```diff
@@ -18,9 +19,11 @@ def execute_tests(
     worker: TestWorker,
     tests: [AbstractTestStatistic],
     rvs_store: IRvsStore,
+    result_store: IResultStore,
```

Comment:
Seems like it is needed to add a ```result_store``` parameter to ```execute_test_step```  function and pass it to ```execute_tests``` function call

Reply:
Fixed